src/content/docs/mobile-monitoring/new-relic-mobile-android/get-started/new-relic-mobile-and-dt.mdx

```diff
@@ -0,0 +1,43 @@
+---
+title: New Relic mobile monitoring with distributed tracing
```

Comment:
Should this title mention android?

Reply:
@bradleycamacho, although this doc lives in the Andoid directory structure, it's eventually going to contain both Android and iOS information once the latter is supported with distributed tracing. When iOS support comes out, I will link to this from that section, too. So, I think this is OK!